### DISCUSSION TITLE
How to make notify like dunst and change font size

### DISCUSSION THREAD

**Discussion by wildan-pratama** [ErikReider/SwayNotificationCenter#242] [Q&A] [answered]
i want make my notifications format like my dunst before and change the fonst size lite this

https://user-images.githubusercontent.com/84622086/230526336-0a2d3806-c6a6-4172-8687-56342e7de4ee.mp4

**Comment by ErikReider** [comment, marked as answer]
Font size can be changed in the CSS. The notification title only supports drawing on one line atm. The body supports multiline (`notify-send 'Test notification' 'text'`). Hope this helps :)
